pybloom_live/test_pybloom.py

```diff
@@ -68,6 +68,17 @@ def _run():
             bloom_one.union(bloom_two)
         self.assertRaises(ValueError, _run)
 
+    def test_union_scalable_bloom_filter(self):
+        bloom_one = ScalableBloomFilter(mode=ScalableBloomFilter.SMALL_SET_GROWTH)
+        bloom_two = ScalableBloomFilter(mode=ScalableBloomFilter.SMALL_SET_GROWTH)
+        chars = [chr(i) for i in range_fn(97, 123)]
+        for char in chars[int(len(chars) / 2):]:
+            bloom_one.add(char)
+        for char in chars[:int(len(chars) / 2)]:
+            bloom_two.add(char)
+        new_bloom = bloom_one.union(bloom_two)
+        for char in chars:
+            self.assertTrue(char in new_bloom)
```

Comment:
Yes, we can test the resulting count. However, this makes only sense if the two bloom filters do not overlap which is the case here. I'm just wondering what should be done if the bloom filters do overlap. In this case, the sum of the individual counts will overestimate the correct total count.

Checking the count value, however, does not make sure the actual union operation is executed correctly. That was the intention of the original test. I'm aware of the false positive issue of bloom filters. Therefore the test does not check for non-existing elements, which might fail due to false positives. However, we should still check for right positives, or?